cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/applications/ApplicationManifestUtils.java

```diff
@@ -163,10 +163,14 @@ private static void asMemoryInteger(JsonNode payload, String key, Consumer<Integ
             } else if (raw.isTextual()) {
                 String text = raw.asText();
```

Comment:
If you put the `toUpperCase()` here, it might read a bit better and not affect any of the parsing.

I'm suggesting:
```
 String text = raw.asText().toUpperCase();
```

and then:
```
               if (text.endsWith("G")) {
                      return Integer.parseInt(text.substring(0, text.length() - 1)) * GIBI;
               } else if (text.endsWith("GB")) {
                      return Integer.parseInt(text.substring(0, text.length() - 2)) * GIBI;
               } else if (text.endsWith("M")) {
                      return Integer.parseInt(text.substring(0, text.length() - 1));
               } else if (text.endsWith("MB")) {
                      return Integer.parseInt(text.substring(0, text.length() - 2));
               } else {
```
later on.